docs/sources/administration/service-accounts/migrate-api-keys.md

```diff
@@ -1,93 +1,37 @@
 ---
 aliases:
-  - about-api-keys/
-  - create-api-key/
-description: This section contains information about API keys in Grafana
+  - ../api-keys/ # /docs/grafana/<GRAFANA_VERSION>/administration/api-keys/
+  - ../about-api-keys/ # /docs/grafana/<GRAFANA_VERSION>/administration/about-api-keys/
+  - ../create-api-key/ # /docs/grafana/<GRAFANA_VERSION>/administration/create-api-key/
+description: This section contains information about how to migrate legacy API keys to service account tokens
```

Comment:
"This section" won't make sense in the contexts where users see the description (search engine results and social media posts)

```suggestion
description: Learn how to migrate legacy API keys to service account tokens.
```